TYPO3.Neos.Kickstarter/Resources/Private/Generator/Configuration/NodeTypes.yaml

```diff
@@ -2,7 +2,7 @@
 # to your pages. You can add as many areas as you like, still only those that are rendered in your TypoScript and
 # Template are visible on your page.
 ---
-#'TYPO3.Neos.NodeTypes:Page':
+#'Neos.Neos.NodeTypes:Page':
 #  childNodes:
 #    'sidebar':
-#      type: 'TYPO3.Neos:ContentCollection'
\ No newline at end of file
+#      type: 'Neos.Neos:ContentCollection'
```

Comment:
Even though this is just a comment, it will lead people to the use of a not (yet) existing node type (see what I mean?) So, please only merge with the actual vendor name change.